mediagalaxy.ro,"HTTPSConnectionPool(host='mediagalaxy.ro', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mediagalaxy.ro', port=80): Read timed out. (read timeout=2)"
medicana.com.tr,"HTTPSConnectionPool(host='www.medicana.com.tr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='medicana.com.tr', port=80): Read timed out. (read timeout=2)"
mediclinic.ae,"HTTPSConnectionPool(host='mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.mediclinic.ae', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
medik-on.ro,"HTTPSConnectionPool(host='medik-on.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='medik-on.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
medikamente-per-klick.de,"HTTPSConnectionPool(host='medikamente-per-klick.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.medikamente-per-klick.de', port=443): Read timed out. (read timeout=2)"
medvestnik.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
medwave.cl,"HTTPSConnectionPool(host='medwave.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee82c4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='medwave.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee90ab80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mefos.hr,"HTTPSConnectionPool(host='mefos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee8a1d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mefos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee8625e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
megashop.hr,"HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.megashop.hr', port=443): Read timed out. (read timeout=2)"
meiji.ac.jp,"HTTPSConnectionPool(host='meiji.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee9ae700>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meiji.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee83bfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
melody4arab.xyz,"HTTPSConnectionPool(host='melody4arab.xyz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee51ceb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='melody4arab.xyz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee51c6d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
men.gov.ma,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mendoza.gov.ar,"HTTPSConnectionPool(host='mendoza.gov.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee6eb910>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mendoza.gov.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee681d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mendozapost.com,"HTTPSConnectionPool(host='mendozapost.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mendozapost.com', port=443): Read timed out. (read timeout=2)"
menlhk.go.id,"HTTPSConnectionPool(host='menlhk.go.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee698910>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menlhk.go.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee7b3b80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menofia.edu.eg,"HTTPSConnectionPool(host='menofia.edu.eg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee871880>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menofia.edu.eg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee83b4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menoinfo.fi,"HTTPSConnectionPool(host='menoinfo.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee625fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='menoinfo.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee4d8a30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
menpan.go.id,"HTTPSConnectionPool(host='menpan.go.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='menpan.go.id', port=443): Read timed out. (read timeout=2)"
mep.go.cr,Exceeded 30 redirects.,Exceeded 30 redirects.
mercadopublico.cl,"HTTPSConnectionPool(host='mercadopublico.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee6983a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercadopublico.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedeea76400>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mercedes.gob.ar,"HTTPSConnectionPool(host='mercedes.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fededb57e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mercedes.gob.ar', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee8a18e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
meteo.lt,"HTTPSConnectionPool(host='meteo.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='www.meteo.lt', port=80): Read timed out. (read timeout=2)"
meti.go.jp,"HTTPSConnectionPool(host='meti.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded108c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='meti.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee441be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metjekutindevut.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
metro.tokyo.jp,"HTTPSConnectionPool(host='metro.tokyo.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fededc58100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='metro.tokyo.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee546fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metrolyrics.com,"HTTPSConnectionPool(host='metrolyrics.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee441a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='metrolyrics.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee9ae7c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
metrouniv.ac.id,"HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='metrouniv.ac.id', port=443): Read timed out. (read timeout=2)"
metsalehti.fi,"HTTPSConnectionPool(host='metsalehti.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee443d30>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPSConnectionPool(host='www.metsalehti.fi', port=443): Read timed out. (read timeout=2)"
mext.go.jp,"HTTPSConnectionPool(host='mext.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee529700>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mext.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fededc571f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.ee,"HTTPSConnectionPool(host='mfa.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded04e970>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mfa.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee8220a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mfa.gov.sg,"HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))","HTTPSConnectionPool(host='mfa.gov.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mfa.gov.sg' doesn't match 'www.mfa.gov.sg'"")))"
mfa.gov.tr,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPSConnectionPool(host='mfa.gov.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mfe.govt.nz,"HTTPSConnectionPool(host='mfe.govt.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mfe.govt.nz', port=443): Read timed out. (read timeout=2)"
mh.gob.sv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mhlw.go.jp,"HTTPSConnectionPool(host='mhlw.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee43f220>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mhlw.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee8a18e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
micheleabruzzese.it,Exceeded 30 redirects.,Exceeded 30 redirects.
michoacan.gob.mx,"HTTPSConnectionPool(host='michoacan.gob.mx', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='michoacan.gob.mx', port=443): Read timed out. (read timeout=2)"
microtelinngatlinburg.net,"HTTPSConnectionPool(host='microtelinngatlinburg.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fededed58b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='microtelinngatlinburg.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded4b2430>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mie.lg.jp,"HTTPSConnectionPool(host='mie.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee6c8640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mie.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedebfb7100>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
migdalor-news.co.il,"HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.migdalor-news.co.il', port=443): Read timed out. (read timeout=2)"
mikmak.co.il,"HTTPSConnectionPool(host='mikmak.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7feded61f160>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mikmak.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7feded6b5fa0>, 'Connection to mikmak.co.il timed out. (connect timeout=2)'))"
milejemplos.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
min-saude.pt,"HTTPSConnectionPool(host='min-saude.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee5a2370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='min-saude.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedeea58e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minbynews.dk,"HTTPSConnectionPool(host='minbynews.dk', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='minbynews.dk', port=80): Read timed out. (read timeout=2)"
minciencias.gov.co,"HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minciencias.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mincyt.gob.ar,"HTTPSConnectionPool(host='mincyt.gob.ar', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded4a9a90>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPSConnectionPool(host='www.argentina.gob.ar', port=443): Read timed out. (read timeout=2)"
mindop.sk,"HTTPSConnectionPool(host='mindop.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='mindop.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fededb57bb0>, 'Connection to mindop.sk timed out. (connect timeout=2)'))"
minec.gob.ve,"HTTPSConnectionPool(host='minec.gob.ve', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee8574f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minec.gob.ve', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fededb57340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineco.gob.gt,"HTTPSConnectionPool(host='mineco.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded4a7040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mineco.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee7cc820>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mined.gob.sv,"HTTPSConnectionPool(host='mined.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee5d74c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mined.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedebec1dc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minedu.gob.bo,"HTTPSConnectionPool(host='minedu.gob.bo', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7feded255940>, 'Connection to minedu.gob.bo timed out. (connect timeout=2)'))","HTTPConnectionPool(host='minedu.gob.bo', port=80): Read timed out. (read timeout=2)"
minedu.sk,"HTTPSConnectionPool(host='minedu.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded0cd6a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minedu.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded2cf850>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mineducacion.gov.co,"HTTPSConnectionPool(host='mineducacion.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded12d250>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mineducacion.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded1722b0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
minetilbud.dk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mininterior.gov.co,"HTTPSConnectionPool(host='mininterior.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fedee9a3340>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mininterior.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fedebf0bd90>, 'Connection to mininterior.gov.co timed out. (connect timeout=2)'))"
ministeriodegobierno.gob.ec,"HTTPSConnectionPool(host='ministeriodegobierno.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ministeriodegobierno.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
ministeriodesalud.go.cr,"HTTPSConnectionPool(host='ministeriodesalud.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded18f1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ministeriodesalud.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedeea05130>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minnanokaigo.com,"HTTPSConnectionPool(host='www.minnanokaigo.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.minnanokaigo.com', port=443): Read timed out. (read timeout=2)"
minsa.gob.pe,"HTTPSConnectionPool(host='minsa.gob.pe', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded18fd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='minsa.gob.pe', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedebe458e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
minsalud.gov.co,"HTTPSConnectionPool(host='minsalud.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded597370>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='minsalud.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedeea05c10>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
mintrabajo.gov.co,"HTTPSConnectionPool(host='mintrabajo.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee4337c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mintrabajo.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded12d8b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mintransporte.gov.co,"HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mintransporte.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minvu.cl,"HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='minvu.cl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
minyu-net.com,"HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='minyu-net.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
mise.gov.it,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
missbloom.bg,"HTTPSConnectionPool(host='missbloom.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7feded1dd670>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='missbloom.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fedee79d3a0>, 'Connection to missbloom.bg timed out. (connect timeout=2)'))"
misumi-ec.com,"HTTPSConnectionPool(host='misumi-ec.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee443fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='misumi-ec.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fededbe8130>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mizuhobank.co.jp,"HTTPSConnectionPool(host='mizuhobank.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee79d130>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mizuhobank.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedebf1b2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mj.pt,"HTTPSConnectionPool(host='mj.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fededbda280>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mj.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded172460>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mju.ac.th,"HTTPSConnectionPool(host='mju.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded092b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mju.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee605730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mku.edu.tr,"HTTPSConnectionPool(host='mku.edu.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded206250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mku.edu.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded0cd760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mladinska.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mlit.go.jp,"HTTPSConnectionPool(host='mlit.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7feded48dac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlit.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded1ddee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mlsi.gov.cy,"HTTPSConnectionPool(host='mlsi.gov.cy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedebeb8e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mlsi.gov.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee482ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mm.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
moa.gov.cy,"HTTPSConnectionPool(host='moa.gov.cy', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moa.gov.cy', port=443): Read timed out. (read timeout=2)"
mobil.nu,"HTTPSConnectionPool(host='mobil.nu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mobil.nu', port=443): Read timed out. (read timeout=2)"
mobilebulgaria.com,"HTTPSConnectionPool(host='mobilebulgaria.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mobilebulgaria.com', port=443): Read timed out. (read timeout=2)"
mobilnisvet.com,"HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mobilnisvet.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moda.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
modenaora.it,Exceeded 30 redirects.,Exceeded 30 redirects.
moe.edu.sg,"HTTPSConnectionPool(host='moe.edu.sg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedec31ff70>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='moe.edu.sg', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee7e2f70>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
moe.go.th,"HTTPSConnectionPool(host='moe.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedee8a1e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moe.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedee7cce80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moe.gov.lk,"HTTPSConnectionPool(host='moe.gov.lk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moe.gov.lk', port=443): Read timed out. (read timeout=2)"
moe.gov.my,"HTTPSConnectionPool(host='moe.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.moe.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mof.gov.vn,"HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mof.gov.vn', port=443): Read timed out. (read timeout=2)"
mofa.go.jp,"HTTPSConnectionPool(host='mofa.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedeeb8ff70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mofa.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7feded27e880>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.bh,"HTTPSConnectionPool(host='moh.gov.bh', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fedebea65e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moh.gov.bh', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedebe64fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moh.gov.jo,"HTTPSConnectionPool(host='moh.gov.jo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='moh.gov.jo', port=80): Read timed out. (read timeout=2)"
moh.gov.my,"HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))","HTTPSConnectionPool(host='moh.gov.my', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'moh.gov.my' doesn't match '*.moh.gov.my'"")))"
mohre.gov.ae,"HTTPSConnectionPool(host='mohre.gov.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mohre.gov.ae', port=443): Read timed out. (read timeout=2)"
